test_files/import_prefixed/import_prefixed_mixed.ts

```diff
@@ -0,0 +1,4 @@
+import * as exporter from './exporter';
+
+let someVar: exporter.TypeExport;
```

Comment:
Comment in these files about what property you're looking for.

Reply:
Done.